src/faces/FacesTool.vala

```diff
@@ -337,7 +337,7 @@ public class FacesTool : EditingTools.EditingTool {
                 Process.spawn_sync(null, argv, null, SpawnFlags.STDERR_TO_DEV_NULL, null, out output);
 
             } catch (SpawnError e) {
-                stderr.printf("Error trying to spawn face detection program: %s\n", e.message);
+                critical("Error trying to spawn face detection program: %s\n", e.message);
                 assert_not_reached();
```

Comment:
You missed one comment here. This is not a programming error, hence I would not assert.
I would rather remember that there was an error in the FaceDetectionJob, exit the function with "return", then check in the callback if the Job had an error and present it to the user with AppWindow.error_message